pyuvdata/uvdata/mwa_corr_fits.py

```diff
@@ -912,12 +912,29 @@ def _get_pfb_shape(self, avg_factor):
         cb_array : numpy array of type float
             Array corresponding to pfb shape for a coarse band.
         """
-        with h5py.File(
-            DATA_PATH + "/mwa_config_data/MWA_rev_cb_10khz_doubles.h5", "r"
-        ) as f:
-            cb = f["coarse_band"][:]
-        cb_array = cb.reshape(int(128 / avg_factor), int(avg_factor))
-        cb_array = np.average(cb_array, axis=1)
+        if mwax:
+            if self.channel_width[0] == 40000:
+                with h5py.File(
+                    DATA_PATH + "/mwa_config_data/mwax_pfb_bandpass_40kHz.h5", "r"
+                ) as f:
+                    cb_array = f["coarse_band"][:]
+            elif self.channel_width[0] == 80000:
+                with h5py.File(
+                    DATA_PATH + "/mwa_config_data/mwax_pfb_bandpass_80kHz.h5", "r"
+                ) as f:
+                    cb_array = f["coarse_band"][:]
+            else:
+                raise ValueError(
+                    "mwax passband shapes are only available for 40 kHz and 80 kHz data; \
+                        resubmit with correct_coarse_band=False"
```

Comment:
Maybe add something here about making an issue on the repo if a user needs it for another channel width?

Also change "data" to "fine channel width" or something similar to make it clear what the exact issue is.